src/addrman.cpp

```diff
@@ -1065,6 +1095,15 @@ int AddrManImpl::CheckAddrman() const
     if (nKey.IsNull())
         return -16;
 
+    int cnt_new{0};
```

Comment:
That would get me a compiler warning `comparison of integer expressions of different signedness`, since `nNew` and `nTried` are `int`. But I'll change the sanity check to compare with the actual totals instead of `nNew` and `nTried`, as suggested by you below.